### PR TITLE
Static and dynamic error pages for wagtail and apache2

### PR DESCRIPTION
**Pull request by akmiller01**
Some of this work was done and already merged into `dev` but evidently not merged downstream. Replicating it here in a hurry for the Alpha.

### CODE REVIEW

File: iati/iati/templates/500.html [hunk 1]
```diff
@@ -1,17 +1,13 @@
-<!DOCTYPE html>
```
Comment (dalepotter) [inline iati/iati/templates/500.html]: As the only differences between this and `iati/iati/templates/404.html`, we should make this into a new template, perhaps called `error-base.html`. Then the `500` and `404` pages can set their own `H1` and `p` content.

**Comment by dalepotter** [comment]
> Some of this work was done and already merged into dev but evidently not merged downstream. 

Does this not mean we can merging `dev` into `add-page-models` would fix it?

File: iati/iati/templates/500.html (hunk 1 above)
Comment (akmiller01) [reply iati/iati/templates/500.html]: @dalepotter Done.

**Comment by akmiller01** [comment]
@dalepotter We were just testing the static error pages in Dev. We will eventually merge them, but this branch has gone pretty far beyond what was in dev already.

**Pull request merged**